mobileportfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Portfolio</title>
+    <link rel="stylesheet" href="mobileportfolio.css">
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
+                
+</head>
+<body>
+    <main class="bg-shapes">
+       <header class="logo-nav">
+            <nav class="navbar">
+                <div class="my-logo">My Logo</div>
+                <a href="#" class="toggle-button">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </a>
+                <div class="hamburger-menu">
+                    <ul>
+                        <li><a href="#">Portfolio</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+
+        <div class="headline">
+            <h2>I'm Mariane <br> Glad to see you &#33;</h2>
```

Comment:
- [optional] you can add the `h2` a class name to use for styling to reduce the font size, so it matches well with the Figma version

Reply:
@tufoinnkuo10 I have both added section tags and modified the h2 font to match figma. Kindly note that this is my last request review attempt.